recipes/_configure_server.rb

```diff
@@ -5,6 +5,9 @@
 #
 # Configures the system server.conf file
 
+default_pass4symmkey = 'changeme'
+default_ssl_password = 'password'
```

Comment:
do these need to be configurable cookbook attributes? Or I suppose that could be a future enhancement to support a non-default ssl keys file/password.